routes/login.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://subtleui.netlify.app/styles/styles.css"
+    />
+    <script
+      src="https://kit.fontawesome.com/12af55b294.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="../styles/styles.css" />
+    <link rel="stylesheet" href="../styles/login.css" />
+    <title>Sneakoholics | Login</title>
+  </head>
+  <body>
+    <nav class="navbar fixed">
```

Comment:
It's good to wrap the main nav in a "header" tag.

Reply:
Thanks for the suggestion Shivansh...made the changes.